integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -985,7 +985,22 @@ public static void createDockerRegistrySecret(
       throws Exception {
 
     ExecCommand.exec("kubectl delete secret " + secretName + " -n " + namespace);
-    String command =
+    String command = null;
+    if (dockerEmail == null) {
```

Comment:
can you add just --docker-email= part conditionally instead of the entire command?

Reply:
Sure.
quicktest on both internal and external jenkins with these changes passed. Please see the result link at description section

Reply:
I mean non-JRF quicktest as well as there are code changes in common code. The links in description are JRF quicktest I guess.

Reply:
Oh, i see. Will do.